docs/moodle.md

```diff
@@ -0,0 +1,136 @@
+### Moodle Admin setup
+
+#### Prerequisites 
+
+Before heading to Moodle, you should get an LTI passport from administrators of the Marsha server to which you want to connect. 
+
+The passport should consist in 2 oauth credentials: a consumer key and a shared secret. 
+
+#### Create external tool preset
+
+In Moodle, logged as an admin, go to `Site administration`:
+
+<kbd>![moodle_lti_select_admin_1](images/moodle_lti_select_admin_1.png)</kbd>
```

Comment:
I don't understand all these `kdb` tags. IMO it's not made to wrap an image.

Reply:
That's what I found in sof to make a frame around images in github markdown... It does what I want : https://github.com/openfun/marsha/blob/3c55b94d5039445a56b1ae6599d22f1b0ee5ff76/docs/moodle.md

Reply:
Maybe use figure instead ? https://developer.mozilla.org/fr/docs/Web/HTML/Element/figure

Reply:
only kbd works... but I agree it's a shitty hack... I didn't pay attention. Will remove :man_shrugging:

Reply:
Regarding this : https://stackoverflow.com/questions/37349314/is-it-possible-to-add-border-to-image-in-github-markdown
maybe we can use a comment `>` ?

Reply:
It works nicely with:
```
<table><tr><td>
  <img src="images/moodle_lti_select_admin_1.png" alt="Moodle LTI select admin view 1"/>
</td></tr></table>
```

What do you think?

Reply:
confirmed with @lunika irl :+1: